src/service/resources-impl.js

```diff
@@ -978,6 +1185,54 @@ export class ResourcesImpl {
    * @private
    */
   discoverWork_() {
+    if (this.intersectionObserver_) {
```

Comment:
Ditto related to a note above. Remeasuring could be important for relayout of some elements. We don't yet use `ResizeObserver` much which would make that simpler. And InOb doesn't fire on all resizes - only when they cause `isInterecting` changes.